synthtool/gcp/partials.py

```diff
@@ -21,9 +21,9 @@
 _DEFAULT_PARTIAL_FILES = [".readme-partials.yml", ".readme-partials.yaml"]
```

Comment:
What's the reason why you didn't choose to add `.integration-partials.yaml` to this `_DEFAULT_PARTIAL_FILES`? 

My first impression is that the change would be simpler.

Reply:
I think customer may want to explicitly use partials in `java.common_templates`.

Even if I added `.integration-partials.yaml` to `_DEFAULT_PARTIAL_FILES`, the logic still needs change because previously only one partial file will be loaded.

Reply:
> the logic still needs change because previously only one partial file will be loaded.

Yes, I think that's valid.

```
    for file in files + _DEFAULT_PARTIAL_FILES:
        partials_file = cwd_path / file
        if os.path.exists(partials_file):
            with open(partials_file) as f:
                result.update(yaml.load(f, Loader=yaml.SafeLoader))
```

> I think customer may want to explicitly use partials in java.common_templates.

I don't feel so. But it's not strong. Think about that (possibility of not needing `partial_files` arguments to  `java_library` and `_load_generic_metadata`) again. If you still feel it's better, then I'm good.

Reply:
I decided not to change the parameter of `java_library` and `_load_generic_metadata` to reduce the complexity of this PR.